### DISCUSSION TITLE
Benchmarking every target

### DISCUSSION THREAD

**Discussion by Balaika** [ropensci/targets#760] [Help] [answered]
Hi,

Thanks for the package. It changes my coding life. :1st_place_medal: 
I would like to know whether it is possible to automatically benchmark every tar_target?

Expected output:
targets : data : processtime  : XXs 
targets : model : processtime  : XXs 
targets : summary : processtime  : XXs

**Comment by wlandau** [comment, marked as answer]
`tar_meta(all_of(c("data", "model", "summary")), seconds)` gives you the runtimes, `tar_visnetwork(label = "time")` displays runtime below the nodes of the graph.

**Comment by Balaika** [comment]
Wow, awesome !
